packages/terra-clinical-result/src/common/other/_ResultError.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import IconError from 'terra-icon/lib/icon/IconError';
+import classNames from 'classnames/bind';
+import styles from './OtherTemplates.module.scss';
+
+const cx = classNames.bind(styles);
+
+const ResultError = (props) => {
+  const {
+    ...customProps
+  } = props;
+
+  const templateClassnames = cx([
+    'system-error',
+  ]);
+
+  return (
+    <React.Fragment>
```

Comment:
You should be able to remove the `<React.Fragment>` wrapper of the span

Reply:
Removed https://github.com/cerner/terra-clinical/pull/574/commits/9eeacc2469da67d5f2d9ad3ce7cc533a407dbf13?file-filters%5B%5D=.js&file-filters%5B%5D=.json&file-filters%5B%5D=.jsx&file-filters%5B%5D=.scss&file-filters%5B%5D=.snap#diff-99f15ff9b73115ee14a2299384936551R18